app/src/main/kotlin/no/nav/k9punsj/felles/dto/FellesDtoer.kt

```diff
@@ -51,8 +51,6 @@ data class OpprettNySøknad(
     val journalpostId: String,
     val pleietrengendeIdent: String?,
     val annenPart: String?,
-    // TODO endre til å bare bruke pleietrengendeIdent, men støtter både barnIdent og pleietrengendeIdent
-    val barnIdent: String?
```

Comment:
Er det ingen som bruker `barnIdent` lenger?
Hvis så, anbefaler jeg å legge på @JsonAlias på `pleietrengendeIdent ` med støtte for `barnIdent `.

Reply:
Nice 👍